cloudify_cli/execution_events_fetcher.py

```diff
@@ -100,9 +113,21 @@ def wait_for_execution(client,
                        include_logs=False,
                        timeout=900):
 
+    # we must make sure the execution and event fetching loops can run at
+    # least once (taking into account the 3-sec sleep here and possible
+    # 1-sec sleep on event fetching)
+    if timeout and timeout < 10:
```

Comment:
If you are taking into account the 3sec sleep + 1 possible sec, where did 10 come from?